src/auth/auth-config.ts

```diff
@@ -714,8 +624,8 @@ export class SAMLConfig implements SAMLAuthProviderConfig {
     this.displayName = response.displayName;
   }
 
-  /** @return {SAMLAuthProviderConfig} The plain object representation of the SAMLConfig. */
-  public toJSON(): SAMLAuthProviderConfig {
+  /** @return The plain object representation of the SAMLConfig. */
+  public toJSON(): object {
```

Comment:
I forget... are we supposed to use `object` or `Object` here? :)

Reply:
`object` is preferred over `Object`. In latest iterations of the tools both are considered problematic, and advocate for `Record<string, unknown>`. But for now we can use `object`.